### PR TITLE
152 user emails

### LINKED ISSUE #152: User emails
## Description
As an admin I want to know the email addresses of my users so that I can contact them whenever necessary.

## Acceptance criteria
- [x] Emails associated with users are stored in the DB
- [x] When creating a user, an email can be entered. 
- [x] An admin can see each users email on the admin-page

### PR DESCRIPTION
**Pull request by Gigset**
## Summary
Users now have a field "email", both in the userform, usertable as well as the database. Emails are checked for being valid.

## Definition of done
- [x] All acceptance criteria are completed.
- [x] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [x] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #152 
email152nr2
email152nr1

### CODE REVIEW

File: src/Client/Pages/Admin/UserForm.razor [hunk 1]
```diff
@@ -56,19 +63,40 @@
...
+        try
+        {
+            var addr = new MailAddress(email);
+            return addr.Address == trimmedEmail;
+        }
+        catch
+        {
+            return false;
+        }
+    }
```
Comment (Gigset) [inline src/Client/Pages/Admin/UserForm.razor:98]: Jag har ingen aning om den här metoden bör ligga här, så ifall ni tycker den hör hemma någon annanstans hojta. :)

File: src/Client/Pages/Admin/UserForm.razor [hunk 2]
```diff
@@ -56,19 +63,40 @@
...
 
+    private bool IsValidEmail(string email)
+    {
+        var trimmedEmail = email.Trim();
+        try
+        {
+            var addr = new MailAddress(email);
+            return addr.Address == trimmedEmail;
+        }
+        catch
```
Comment (martinjonsson01) [inline src/Client/Pages/Admin/UserForm.razor:94]: It's bad practice to catch all exceptions, you should only catch the exceptions you specifically expect!

E.g.
```suggestion
        catch(FormatException)
```

File: src/Client/Pages/Admin/UserForm.razor (hunk 1 above)
Comment (martinjonsson01) [reply src/Client/Pages/Admin/UserForm.razor:98]: Ideally, email validation should be done on both server and client side, and in that case it should be inside of Application. You could create a class `EmailValidator` or something similar there.

**Review by martinjonsson01 — APPROVED** [review]
lgtm 👍 

Just make sure to fix that catch-all!

**Review by Matte02 — APPROVED** [review]
Nice 👍

**Pull request merged**